packages/cli/src/lib/snapshot/pullModel/validation/model.schema.json

```diff
@@ -8,25 +8,25 @@
     "$schema": {
       "type": "string"
     },
-    "resources": {
+    "resourcesToExport": {
       "type": "object",
       "additionalProperties": false,
-      "properties": {
-        "EXTENSION": {"type": "array", "items": {"type": "string"}},
-        "FIELD": {"type": "array", "items": {"type": "string"}},
-        "ML_MODEL": {"type": "array", "items": {"type": "string"}},
-        "QUERY_PIPELINE": {"type": "array", "items": {"type": "string"}},
-        "SEARCH_PAGE": {"type": "array", "items": {"type": "string"}},
-        "SOURCE": {"type": "array", "items": {"type": "string"}},
-        "SUBSCRIPTION": {"type": "array", "items": {"type": "string"}}
+      "patternProperties": {
```

Comment:
Similar as previous comment. The CLI should not impose a list of the resources to include in the SPM.
The platform-client is already here for that.
Also, resource validation is already being taken care in the backend if the user includes an invalid resource in the SPM.